nextflow_schema.json

```
@@ -292,24 +296,58 @@
             "description": "Options specific for the ExomeDepth execution",
             "default": "",
             "properties": {
+                "yamlconfig": {
+                    "type": "string",
+                    "description": "YAML config file to use in BEDGOVCF module"
```

```suggestion
                    "description": "YAML config file to use in BEDGOVCF module",
                    "exists": true,
                    "format": "file-path"
```
Some extra validation parameters :)

---

Added